app/views/ImporterConfirmationView.scala.html

```diff
@@ -29,33 +35,46 @@ <h1 class="govuk-panel__title">
             @messages("confirmation.heading")
         </h1>
         <div class="govuk-panel__body">
-            <div id = "entryNumber">@messages("confirmation.entryNumber")</div>
+            <div id = "entryNumber">@messages("confirmation.caseId")</div>
             <strong>@reference</strong>
         </div>
         <br>
     </div>
 
-    @p(Html(messages("confirmation.p1")), id = Some("p1"))
-
-    <img id = "icon" class="print-this__icon" src="https://www.tax.service.gov.uk/assets/4.9.0//images/icon-print.svg" alt="Print Icon"/>
-    <a id = "printSave" class="print-this__link govuk-body" href="javascript:window.print()">@messages("confirmation.printSave")</a>
-    <a id = "printSaveRestOfMessage" class="print-this__link govuk-body">@messages("confirmation.printSaveRestOfMessage")</a>
+    @if(isSingleEntry) {
+        @p(Html(messages("confirmation.singleEntry.p1", data.entryDetails)))
+    } else {
+        @p(Html(messages("confirmation.bulkEntry.p1", data.importerName, data.eori)))
+    }
 
     @h2(messages("confirmation.whatHappensNext"), id = Some("whatHappensNext"))
-    @p(Html(messages("confirmation.p2")), id = Some("p2"))
-    @p(Html(messages("confirmation.p3")), id = Some("p3"))
 
-    @h2(messages("confirmation.helpImproveService"), id = Some("helpImproveService"))
+    @if(isPayByDeferment) {
+        <p class="govuk-body">@messages("confirmation.whatHappensNext.deferment.p2")</p>
```

Comment:
Is there a reason for not using the component.p helper in this line? Ditto for a few other places too.